Serverdb: add users roles

Cedric921 opened the pull request.

Bam92 committed:
style: add the /users routes on the navbar

Cedric921 committed:
fix: fix uer error on lading

Cedric921 committed:
feat: update routes to add users

Cedric921 committed:
fix : fix merge confict

Cedric921 committed:
Merge branch 'main' into serverdb

Cedric921 committed:
fix: fix add user error

Cedric921 committed:
fix: fix merge error

Bam92 commented:
- del unnecessary comments (mostly in `controllers/admin.js`)
- add a summary of the three added roles

Cedric921 committed:
fix: fix add user error

Cedric921 committed:
fix: fix confirm password

Cedric921 commented:
- I have already deleted unnecessary comments
- for users, we have 3 roles
       1. Super admin: he can add, edit and delete users, add, edit and delete students, he can also add attendance;
       2. Admin: can add, edit and delete students and add attendance
       3. Attendance: he can only add attendance and see statistics
 - I have fixed the confirm password on add user to be necessary
 - I have also fixed the navbar

Cedric921 committed:
feat: fix pagination with sequelize
